barretenberg/cpp/src/barretenberg/flavor/goblin_ultra_recursive.hpp

```diff
@@ -114,6 +114,7 @@ template <typename BuilderType> class GoblinUltraRecursiveFlavor_ {
          */
         VerificationKey(CircuitBuilder* builder, const std::shared_ptr<NativeVerificationKey>& native_key)
```

Comment:
Question: is there any point in keeping the other constructor here? It seems dangerous to not set up the verification key with the pcs_verification_key...

Reply:
Feel free to investigate and resolve here though it may make sense to resolve in a broader PR related to the non-use of copy constructors in the Pg code.